__tests__/app.test.js

```diff
@@ -200,9 +200,30 @@ describe('GET /api/articles/:article_id', () => {
           votes: 100,
           article_img_url:
             'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700',
+          comment_count: 11,
```

Comment:
Ideally we should not be going back to alter previous tests - it's something worth thinking about in the future. How can we write tests that won't break if we decide further down the line to add more features to the endpoint?

Reply:
I've updated the comment_count property and updated the test to use `.toMatchObject` instead of `.toEqual`

I could've used that previously to make sure this test can dynamically handle new features being added!